aws/table_aws_config_delivery_channel.go

```diff
@@ -0,0 +1,205 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/configservice"
+	"github.com/aws/aws-sdk-go-v2/service/configservice/types"
+
+	configservicev1 "github.com/aws/aws-sdk-go/service/configservice"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsConfigDeliveryChannel(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_config_delivery_channel",
+		Description: "AWS Config Delivery Channel",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"name", "region"}),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"NoSuchDeliveryChannelException"}),
+			},
+			Hydrate: getConfigDeliveryChannel,
+			Tags:    map[string]string{"service": "config", "action": "DescribeDeliveryChannels"},
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listConfigDeliveryChannels,
+			Tags:    map[string]string{"service": "config", "action": "DescribeDeliveryChannels"},
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(configservicev1.EndpointsID),
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name of the delivery channel.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "arn",
```

Comment:
Could you please remove the `arn` column, as the API does not provide ARN details?